### DISCUSSION TITLE
cn/2024/12/22/baiwan/

### DISCUSSION THREAD

**Discussion by giscus** [hongtaoh/hongtaoh.github.io#201] [Announcements]
# cn/2024/12/22/baiwan/

https://hongtaoh.com/cn/2024/12/22/baiwan/

**Comment by hongtaoh** [comment]
@earfanfan 等你来吐槽。

**Reply by earfanfan** [reply]
不论这是畅想还是幻想，我有兴趣加入，请你考虑接受。我之所以有兴趣是因为工作中会接触到 RAG 项目（知识问答），我现在就在琢磨怎么用 Ragas 框架去评估 RAG 项目的使用效果，我们是调用别人的 API，需要测试 AI 的一些能力比如文档解析、检索召回、语言生成、上下文理解等，可是能够调整的地方实在是太有限了。另外 AI 的幻觉也是我感到很难解决的问题。我倒是真的好奇 AI 是怎么训练出来的，到底可以怎么调。

其实这个想法似乎是你最近多番思考的一个综合成果，也可能是我刚好只看了几篇，能看到里面有，你认为知识传播的载体用文本更高效，AI 本身可以解决非文学类的翻译问题，你学过的一些标准课程只讲怎么做、不讲为什么这让你反感等等。

我真心地赞美这个想法，可我也真心地感到它与现实世界相比仍有些地方太悬浮。如果让一个人来学习这些东西，此人必然需要评估自身的学习意愿是兴趣驱动还是利益驱动、学习能力是否足够覆盖整个过程、学习成本上除了付出时间还需要够格的硬件配置，那么推出的教程本身既是吸引也是筛选。可是在这些以外，很多人好像都很不清醒，无意识地深陷生活的泥沼，所以最开始的辩证就需要想得再强而有力一些，即为什么要彻底掌握 AI？ 即便这些人能够自己训练出一个 GPT2，现在国内那么多免费的 AI 为何不去用，而一个 GPT2 又能有什么用呢，还是说只要能训练一个 GPT2 就一定能快速跟上 AI 发展的时代进度（PS这块细节我倒是不懂，训练的语料到哪里能搞到？）。

关于目标群体，你原先所想的我倒是理解，你想要让零基础的人学会才想到让“普通中国人”来学，可是你似乎并不了解真正的“普通中国人”过的是什么日子，为了生存就要倾尽全力的人确实是零基础，但也许是最不可能去学这些的人。应该是那些还有余力对抗生活的人才有可能愿意去学。

**Reply by hongtaoh** [reply]
>不论这是畅想还是幻想，我有兴趣加入，请你考虑接受。

很感谢袁姐捧场！目前来说，你这个精神支持比实际参与给我的鼓舞更大。目前我能做的是自己先达到我说的水平，还有很长的路要走。

>其实这个想法似乎是你最近多番思考的一个综合成果，也可能是我刚好只看了几篇，能看到里面有，你认为知识传播的载体用文本更高效，AI 本身可以解决非文学类的翻译问题，你学过的一些标准课程只讲怎么做、不讲为什么这让你反感等等。

是的，现在的很多教程门槛太高了。我自己知道我只是一个平庸的科研工作者，我的优势在于我喜欢把复杂的东西讲给不会的人，所以我决定拾人牙慧，去讲别人的东西。

>我倒是真的好奇 AI 是怎么训练出来的，到底可以怎么调。

我没有自己从零开始手写语言模型的训练代码，但我估计你自己写一下[基本的神经网络](https://hongtaoh.com/cn/2023/04/26/nn/) 大概就知道那些参数是什么意思、到底怎么调。

你提到的驱动以及筛选，很有道理。

>为什么要彻底掌握 AI？

我不会去说服别人去学，有点像姜太公钓鱼。如果学 AI 还需要我去说服，那说明 AI 的影响力还没有足够大。

>即便这些人能够自己训练出一个 GPT2，现在国内那么多免费的 AI 为何不去用，而一个 GPT2 又能有什么用呢，还是说只要能训练一个 GPT2 就一定能快速跟上 AI 发展的时代进度（PS这块细节我倒是不懂，训练的语料到哪里能搞到？）。

很好的问题。我目前的观察是，如果一个人自己爬取数据、清理数据，从零开始写底层代码，自己训练出 GPT-2 级别的模型，这个水平已经达到甚至超过大多数美国 CS PhD 学生。

你提到的国内那么多免费的 AI 为什么不用。这就像我们为什么不直接买美国的芯片和飞机，非要自己造呢？使用 API 接口谁都可以，但这样没办法走在 AI 最前沿，也就找不到高薪工作。

>可是你似乎并不了解真正的“普通中国人”过的是什么日子，为了生存就要倾尽全力的人确实是零基础，但也许是最不可能去学这些的人。应该是那些还有余力对抗生活的人才有可能愿意去学。

很深的思考。首先，解决贫困不是我能考虑的问题。其次，你提到的我不了解真正的普通中国人过的什么日子，这一点我不同意。我觉得对绝大多数我的同龄人来说，我个人以及我家庭的收入是处在比较低的水平的。这很大程度不是有没有钱的问题，而是愿不愿意投资自己的时间和精力的问题。如果一个人没有工作，全职做一年的外卖员，他/她挣得钱足够找一个偏僻的地方，全身心学习三到五年的时间。

**Reply by earfanfan** [reply]
当我写“为了生存就要倾尽全力的人”时，联想到的是和我年纪差不多的农民工群体，我们村子里就有很多，我还上学的年纪他们/她们就遵从长辈的意愿结婚生娃了，现在才三十出头的年纪就已经被生活摧残得和我印象中他们的爸爸妈妈一个样子了。这个群体身上的负担太重，已经没有办法投资自己的时间了。

>如果一个人没有工作，全职做一年的外卖员，他/她挣得钱足够找一个偏僻的地方，全身心学习三到五年的时间。

如果是没有成家的，而且能够远离原生家庭的催逼的，可以用破釜沉舟的勇气远离社会时钟带来的焦虑，走完了这三到五年的学习时间以后，再次进入就业市场时，可能还是会因为学历、年龄，甚至空白期、有技术而没有经验再遭受到严重的就业歧视。

唉，是我不好，我太悲观了，不过好在郝糊涂你那么倔强，一定不会受到我的影响的。我很期待这个项目真正出现的那一天。如果非要说这一天什么时候到来，我想应该是确确实实有很多人认识到这方面的工作存在大量的缺口，或者国家政策上给予些什么有效资源倾斜过来。

**Reply by hongtaoh** [reply]
你提到的农民工群体受到生活的重压，这一点我非常同意。这不是我能改变的。

实话讲，虽然我没有任何歧视，但农名工群体不会成为百万 AI 的主体，甚至可能连 1% 都不到。我猜想的主体大概是高中在读 (5% )+大专在读 (5%) +本科在读 (40%)+研究生在读 (15%)+全职工作(20%)+自由职业(10%)+其他(5%)。也就是说学生群体的比例会大于 60%。

>如果是没有成家的，而且能够远离原生家庭的催逼的，可以用破釜沉舟的勇气远离社会时钟带来的焦虑，走完了这三到五年的学习时间以后，再次进入就业市场时，可能还是会因为学历、年龄，甚至空白期、有技术而没有经验再遭受到严重的就业歧视。

我能预想到这个困难。所以我在考虑成立 AI 咨询公司，学历越低、专业越不匹配、年龄越高、经验越少的优先录用，前提是完成全部学习课程、通过测验、通过面试。对这一部分群体来说，这本来也是一场赌博，赌资是自己的时间、精力、脑力。其实对我来说，来美国读书，之后选择学 CS，我也在赌博，堵上了自己的青春，放弃了一部分人生体验（恋爱）和收入，希望跳出和我同背景的人所在的工作环境和视野，但最后非常可能失败。

>我很期待这个项目真正出现的那一天。如果非要说这一天什么时候到来，我想应该是确确实实有很多人认识到这方面的工作存在大量的缺口，或者国家政策上给予些什么有效资源倾斜过来。

我觉得 00 后、10后、20 后、30 后可能是最后几代有机会、有兴趣学人工智能的人。越往后，门槛会越来越高，越来越难学，大部分人会直接放弃，选择在 AI 这个黑盒子里生活，而不是操控它、改变它。

**Reply by earfanfan** [reply]
>我觉得 00 后、10后、20 后、30 后可能是最后几代有机会、有兴趣学人工智能的人。越往后，门槛会越来越高，越来越难学，大部分人会直接放弃，选择在 AI 这个黑盒子里生活，而不是操控它、改变它。

然而三水和我不在这个年龄范围内，却正好是处于学习能力很强的阶段，也是很有兴趣的那波人呐。

其实如果不从最底层的数学和统计学的知识开始学，而是从中层的应用开始学的话，门槛反而是越来越低的，比如我知道有许多平台比如 coze（中文名扣子,官方介绍这么说的：是新一代 AI 应用开发平台，无论用户/开发者是否有编程基础，都可以在扣子上快速搭建基于大模型的各类智能体，并发布到各个渠道），这也是一种操控。

另外，其实你我讨论 AI 竟然从未讨论过提示词这个东西，三水倒是在说人类是传声筒，在写代码这方面，我感觉 AI 很像是一个能力超强、学习效率超快的业务新手，我描述的事项越详细，它写出来的代码越能直接用。另外，我最近接触到好些 RAG 开源项目，比如RagFlow 支持流程编排，要是让 AI 做翻译，可以让第一个 AI 先直译，然后让第二个 AI 在前者的基础上再意译；还有 Qanything ，这是网易有道出的，在文本片段的检索召回步骤做了重排序，然后大家都发现重排序好用，好多项目就都加上了重排序。说起来，这样的使用也是一种操控。

我在这里写下的第一句话，你要记得哦，等到你真的开始做这个项目的时候，我应该已经积攒了不少在工业界鼓捣 AI 的经验了，说不定将来能对这个项目有用。

>但最后非常可能失败。

讲真，我真的觉得你一定会不会失败。看起来，你很想要财富和女人，将来你一定会得到的。

**Reply by earfanfan** [reply]
忽然想到，郝糊涂的想法不就是以前列弛写《庄子》的时候提到的“物物而不物于物”，我还记得问过三水，三水的答案好像是该吃饭吃饭该吃面吃面，就行了。

**Reply by hongtaoh** [reply]
>然而三水和我不在这个年龄范围内，却正好是处于学习能力很强的阶段，也是很有兴趣的那波人呐。

没把 90 后、80 后放进去是因为如果 30 后是最后几代，那之前的人肯定包括。

你提到的这些依然是在使用 AI 的层面，不是创造 AI。使用这些工具和我在文中提到的用 ChatGPT 做海报和 PPT 没有任何区别。百万 AI 的目标应该是让学生有能力独立研发出类似 coze 和 qanything 的平台和工具，而不是单纯做使用者、调包侠、调参师。

不要误解我，我没有轻视这些工具和使用这些工具的用户，我自己都还远未达到我想让我未来的学生达到的水平。

>我在这里写下的第一句话，你要记得哦，等到你真的开始做这个项目的时候，我应该已经积攒了不少在工业界鼓捣 AI 的经验了，说不定将来能对这个项目有用。

寡人尽量。我很怕给别人承诺，我怕我忘了。如果我非要说「一定」，那我压力就很大了，因为我真的有可能忘，未来五年我要每天提醒自己才能做到不忘。

>讲真，我真的觉得你一定会不会失败。

凡姐好像 ChatGPT。它也是这么鼓励我的。

>你很想要财富和女人，将来你一定会得到的。

扑哧，你这个跳跃有点大。你从哪看出来我「很想要」，而且是财富+女人。凡姐是在感受能量吗？

**Reply by earfanfan** [reply]
>你提到的这些依然是在使用 AI 的层面，不是创造 AI。使用这些工具和我在文中提到的用 ChatGPT 做海报和 PPT 没有任何区别。百万 AI 的目标应该是让学生有能力独立研发出类似 coze 和 qanything 的平台和工具，而不是单纯做使用者、调包侠、调参师。

>不要误解我，我没有轻视这些工具和使用这些工具的用户，我自己都还远未达到我想让我未来的学生达到的水平。

今年有一天我溜达到 COS 论坛去翻到了一个帖子，里面在讨论文档形式的优劣，益辉发言他相信未来一定是 html 的天下，我看到这里时用我的方式理解了他，我想到当时他在开发他的 R 包，其实最需要的并不是对某事物的多元认知，而是最需要相信他本来愿意相信的，这样才有信念继续去做想要做的事。这里我对你的理解也是类似，你也需要相信你相信的，将来就按照自己的想法去做就行。

>寡人尽量。我很怕给别人承诺，我怕我忘了。如果我非要说「一定」，那我压力就很大了，因为我真的有可能忘，未来五年我要每天提醒自己才能做到不忘。

额，因为害怕忘记所以要每天提醒，这看起来很不健康的样子。或者你可以换个角度看待，这并不是一种只需要你来付出的单向承诺，而是一种双向的约定，你有记得的责任，而我也有提醒的义务。如果真的到了项目启动的那天，你会昭告天下，我会看到，那么我会溜达过来问你“喂，你忘记我的存在了吗？”，由于你本来就习惯向我说抱歉，你就会对我说“抱歉抱歉，我真的忘记了”，我会说“好吧，那现在想起来了么，让我也加入吧”，你会说“好嘞，俺们一起”，这样就行了。所以对待双向约定可以轻松很多，约定的时候就记在心里，下一秒就双双忘记，你也忘记，我也忘记，将来再说起的时候记得曾经有过这个约定就行了。

>你从哪看出来我「很想要」，而且是财富+女人。凡姐是在感受能量吗？

俺干活去也，以后再与你细说。

**Reply by hongtaoh** [reply]
好的大耳朵凡凡，如果真有项目启动那一天，我如果还记得，会跟你说一声。大概率你也会从我的网站上看到。我感觉最起码是五年之后的事情了。

>俺干活去也，以后再与你细说。

说的我好怕。财富的话我并没有太大的渴望。女人的话，更多的是为了满足没有满足的愿望，至于是不是发自内心真的想要，以及得到之后是不是会失去兴趣，我暂时不知道。

>这里我对你的理解也是类似，你也需要相信你相信的，将来就按照自己的想法去做就行。

可能是吧。如果一件事自己都不相信，那做起来会很困难。

**Reply by earfanfan** [reply]
>我感觉最起码是五年之后的事情了。

我接受，俺明白。本口头投资者对你这个项目发起者没有撒要求，你就好好活着，爱护好自己的身体和心灵就行了，反正来日方长，皆有可能。

**Comment by earfanfan** [comment]
@yufree

三水，你好，俺们呼唤你，快来看看俺们的幻想。

>我是不介意世界上出现另一个我的，但世界也许很介意。

我刚瞅了瞅你写的世另我，为撒世界会介意？另外你只投喂了104篇自己写的博客，也就是说如果问的问题是超出你的博客内容的，AI 可能会有幻觉使用“通用”能力去回答，那么它始终不会是你。ps，我对 AI 的理解很浅，我理解的“通用能力”是指 AI 本身训练了几十 TB 的语料形成的能力。

**Reply by earfanfan** [reply]
又及，三水，其实我觉得现在的 AI 和 “人类重生”还有很长的距离。我觉得，人类凭借五感（视觉、听觉、嗅觉、味觉、触觉）来感知和识别世界上的信息，运用大脑做加工处理和分析，运用语言文字来传递信息，而现在的大语言模型会受限于语言文字本身的模糊特性，无法确保“信息”的百分百传递。真正的“人类重生”也许还要更加突破想象力一点，更遥远的 AI 可以跳出人类语言的框架束缚，自己去自然界里寻找新的启发，只是你我未必看得到。

**Reply by yufree** [reply]
这个项目我现在没法评价，总感觉后面会出现非transformer的构架，不过人能理解到构架基本也到头了，里面的细节理解可能超出生理限制了。
我用的lora微调，本质上就是用一个小矩阵给原来参数加权而不是直接重调所有参数，所以超出博客内容部分权重自然继承了原有模型。不过这里的问题在于人的语言个性其实很难定义，大部分内容是趋同的，你问一些原则性问题大多数人都是复读机而已，所以我不担心这玩意是不是我，因为我本身也是各种复读机叠加后的一种。104篇大概1M文本量，而微调是有效果的，本质上应该是我在通用语言模型上屏蔽或增强了一些参数的权重，这多少也算有了个性输入了。这个过程其实文本量质量要比大小更关键，例如一个人读了某一本书或某篇文章突然明白了很多事，书或文章本身文本量不必很大，但能起到激活参数作用就可以。建议你亲自动手去调几个3B或5B小模型感受一下，适当引导还是有效果的，最初的大文本量训练更像是九年义务教育，属于给参数初始化。
幻觉这个问题我现在更倾向于认为这是推理功能的初级表现，很多编出来的文献看题目我感觉确实应该有人做过了但现实却没有，是被完型填空进来的，那么是否有可能在某个平行宇宙里基于人类语言认知能力确实写过这样的文献呢？大语言模型给出的是他认为应该存在的，而现实世界不存在更多说明我们这个世界只是众多选择的一个，这些不同的世界背后共享一个理论上的语言认知能力。当然，单纯训练不足更有可能，后面的训练如果想更符合这个世界，是要给语言模型更多对外交互能力接口的。
数字重生不必要很精确，只要一部分人看不出区别就可以，考虑到人认知能力的方差，不会很遥远。现在语言模型接入自动化实验机械臂已经是现实，研究人员给出实验目的与收集到的样本，是可以自动生成实验设计来做实验出报告的，后面只需要让语言模型自己想有意思的科学问题然后给他限定实验资源，然后给他们记忆学习系统（也就是根据实验结果自动微调认知模型），他们是完全可以取代人类做科研的，只是人类的身份将从复读机变成传声筒而已。

**Reply by hongtaoh** [reply]
@earfanfan 

>只是你我未必看得到。

你肯定能看到。我估计用不了 30 年，世界上满大街都是人形机器人。

>大语言模型给出的是他认为应该存在的，而现实世界不存在更多说明我们这个世界只是众多选择的一个

有一种看科幻小说的感觉。

>后面只需要让语言模型自己想有意思的科学问题然后给他限定实验资源，然后给他们记忆学习系统（也就是根据实验结果自动微调认知模型），他们是完全可以取代人类做科研的

这一点我是完全同意的。昨天我还和朋友在讨论，我觉得再过几十年，arxiv 上的最新论文十有八九是人工智能写的。你提到的大模型推测这个世界应该存在但是不存在的论文，人工智能也许会去做。但目前我觉得应该只能做人文和理工科，至于社会科学，需要用人做实验以及做调查问卷，这个比较难。不过到了那个时候，人的地位也许比机器低，研究人类社会已经没有什么意义了。

**Reply by earfanfan** [reply]
@yufree 

>那么它始终不会是你。

三水你写的这些，我也刚刚好一字一句都看得明白。不过我在写下这一句时，所想的是另外一个方面。我觉得三水好像是那种从小到大都很理智的人，博客行文风格也是里里外外都很理智，可我真的莫名其妙地感觉到三水你的内心很柔软。这个柔软不好量化，额，就用对不起他们的方式来比喻一下吧。现在的我所感受到的是，倔强的郝糊涂很像是生活里没有生活的大木头，而迷茫的湘云很像是心里没有心的大石头，相比而言，总是很理智的三水却是生活里有生活、心里有心的一汪海水。以往我溜达到你的博客里留言的那些拍马屁的文字，都是我真心想要赞美你的话。正是因为我的出发点是 AI 没有人性，所以觉得三水 AI 始终不会是人类三水。

>建议你亲自动手去调几个3B或5B小模型感受一下

以前我没有想过，不过以后我会记得要去这样尝试感受一下的。

>幻觉这个问题我现在更倾向于认为这是推理功能的初级表现

我现在接触到的 RAG 项目也是明显觉得 AI 在推理和计算方面好像还差点火候，有时候会出现计算过程正确但最终结果错误的答案，有时候问某某标准会给出一个范围但答案不够完整，还有些时候我都怀疑 AI 是不是成精了非要坚持给出它想给出的答案，但那个答案又确实是我们所在的现实世界里不存在的，也就是三水你说的可能只存在于平行世界的。

>这些不同的世界背后共享一个理论上的语言认知能力。

我明白你要表达的展开来讲的话就是，不同的世界可能会出现不同的语言，每个世界里的“人”要运用那些语言去传递信息，但传递的信息在刨除了语言这种载体后本质上的内容可能是一样的，就比如数学和物理学上的那些经典定理，像是牛顿三定律、万有引力那些。如果再科幻地想象一下，说不定只是我们这些地球上的人类才用语言文字来传递信息呢，在更高维度的世界里，还有更厉害的形式更高效的信息载体。

>后面的训练如果想更符合这个世界，是要给语言模型更多对外交互能力接口的。

我有这样想过，在人类的五感当中视觉和听觉更发达是因为不需要实际接触就能看见和听见，人们可以轻易地根据看见和听见的事物划分标准，有了标准就会有好坏差异，这样原先充满了不确定性的世界就可以借由一些确定性的标准作为工具来进行衡量，人类才开始发现世界上存在的规律，并利用掌握的规律来发展自身、谋求生存。换句话说，另外三种感官需要实际接触才能评判，而语言本身传递的其实是虚空的信息，好像很难像发展视觉和听觉的路子那样建立统一标准、划分好坏差异。等等，难道从环境科学的角度来看，嗅觉、味觉、触觉也是可以量化的么？

又另，我想象不出来符合世界是要符合世界的什么，不过如果是贴合人类特色的话，我想到每个人的认知总是受限于自身的经验知识，而……而我们国家的 AI 有时候竟然也给我一种意识形态领域也有限制的感觉，唉，扯远了。

>不过这里的问题在于人的语言个性其实很难定义，大部分内容是趋同的，你问一些原则性问题大多数人都是复读机而已，所以我不担心这玩意是不是我，因为我本身也是各种复读机叠加后的一种。

道理是这个道理，我也认同，不过你每次提到复读机我都会觉得有点莫名其妙的搞笑。

**Reply by earfanfan** [reply]
说到复读机、传声筒、二极管这几个词，我刚问了一下比我小十岁的堂妹依依，我问“依依，你知道复读机吗”，依依反问“是真的复读机还是指人是复读机啊”。原来依依小时候从没见过真的复读机，只见过手机里自带的会说话的汤姆猫，但她也知道“人是复读机”这种说法，当然她只在教材上看到过传声筒，也没见过真的，二极管倒是见过，不过她统统都知道把人比作这些物体的说法。

我好像知道为撒每次三水说人是复读机而已会被戳中笑点了，因为我小时候见过真正的复读机，所以这个词语在我脑中出现时总会浮现出它的外形，进而闪现出将人类身体和复读机外形结合的画面。

再说到传声筒，这好像是更古老的发明，但是又被当今时代的人们赋予了新的意象，哈哈，人类的创造还挺有趣的，在语言上将传声筒的形与象分离，再将象与新的意结合，这样的话好像可以创造出有限的新意象，又给人无限的幻想的感觉。

**Comment by hongtaoh** [comment]
@earfanfan 我看到了 [https://www.datawhale.cn/](https://www.datawhale.cn/) 和我想做的很像。有很多开源贡献的人。已经很好了。目前看的话，最大的问题是因为是许多人贡献的，所以学习路径不太连贯。另一个问题是对于真正的小白并不友好。

**Reply by earfanfan** [reply]
这种模式已经存在我不觉得稀奇，不过鲸鱼的路线偏向于应用，我看网页上 AI 通识课的口号写的是“有交流、能应用、会实战”，和你的初衷不同。

现在 AI 发展速度太快，我已经跟不上了，前有 deepseek 后有 manus，实不相瞒我现在开始看小红书了，看起来内容还是大多是人写的，要是到了 AI 生成内容十分泛滥的那一天，我都还不知道要如何适应。

**Reply by hongtaoh** [reply]
确实变化太快了，日新月异，不过我依然没有知识焦虑，至少目前不明显。

**Reply by earfanfan** [reply]
莫西莫西，你看清楚点，我表达的不是知识焦虑，而是设想一旦人类世界的生成内容被 AI 裹挟……

等等，为了避免误会，你说的知识焦虑具体是指什么？

**Reply by hongtaoh** [reply]
>你说的知识焦虑具体是指什么

怕错过风口。

我没有看错，我没说你有知识焦虑，我只是说我目前没有。

**Reply by earfanfan** [reply]
明白了，是我的锅，我没说清楚，我说的不知道如何适应本来想表达的是这个方面：一旦 AI 生成的内容开始泛滥，而 AI 又发展出与人独立的一些东西，我这个小小个体所见所得也会因为接触频率而被改变，我所不知道如何适应的是，原先人生成的有人味，以后 AI 生成的没有人味，可能会演变成，我是旧时代的人，不知如何适应新时代的产物。

当然啦，这其实也可以算是一种微末的担忧，也可以说是一种焦虑吧。

**Reply by earfanfan** [reply]
>怕错过风口。

我倒是也没有这种焦虑，正相反，AI 消息太多变化太快，我对这块的新事物的好奇心已经迅速耗尽了，而我的同事有的也觉得快了、前后脚，而也有一些依然对此保有极大的热情。

**Reply by hongtaoh** [reply]
我在[这里](https://www.ruanyifeng.com/blog/2025/03/weekly-issue-340.html) 看到[基因改造的老鼠](https://www.livescience.com/animals/land-mammals/colossal-creates-woolly-mouse-in-new-step-towards-mammoth-de-extinction)。我感觉离基因改造人类已经不远了。未来有钱人的后代估计都是高大、威猛、美丽、帅气、高智商、高情商、高灵商、健康、长寿、勇敢的，这个估计比 AI 的影响要大，如何和 AI 结合起来，影响更大。
